File: 3019-furthest-point-from-origin/furthest-point-from-origin.py
class Solution(object):
    def furthestDistanceFromOrigin(self, moves):
        l, r, res = 0, 0, 0

        for m in moves:
            if m == 'L':
                l += 1
            elif m == 'R':
                r += 1
            else:
                res += 1

        return res + abs(l - r)        

File: 3019-furthest-point-from-origin/furthest-point-from-origin.java
class Solution {
    public int furthestDistanceFromOrigin(String moves) {
        int l = 0, r = 0, res = 0;

    for (char m : moves.toCharArray()) {
        if (m == 'L') {
            l++;
        } else if (m == 'R') {
            r++;
        } else {
            res++;
        }
    }

    return res + Math.abs(l - r);
} }
        
    
